src/main/java/dev/unnm3d/rediseconomy/currency/migration/SqlCurrencyMigration.java

```diff
@@ -0,0 +1,96 @@
+package dev.unnm3d.rediseconomy.currency.migration;
+
+import dev.unnm3d.rediseconomy.RedisEconomyPlugin;
+import dev.unnm3d.rediseconomy.config.Settings;
+import dev.unnm3d.rediseconomy.currency.Currency;
+import dev.unnm3d.rediseconomy.currency.CurrencyMigration;
+import io.lettuce.core.ScoredValue;
+
+import java.sql.*;
+import java.util.*;
+import java.util.logging.Level;
+
+public class SqlCurrencyMigration extends CurrencyMigration {
+
+    private Settings.SqlMigrateSettings sql;
+    private Connection connection;
+
+    public SqlCurrencyMigration(RedisEconomyPlugin plugin, Currency currency) {
+        super(plugin, currency);
+    }
+
+    @Override
+    public String getProvider() {
+        return "SQL DATABASE";
```

Comment:
can you explain this please?

Reply:
Instead of returning a economy provider name like offline player migration, the SQL migration doesn't have that